packages/ui/src/theme/__tests__/defaultTheme.test.ts

```diff
@@ -78,6 +78,14 @@ describe('@aws-amplify/ui', () => {
         --amplify-components-badge-large-font-size: var(--amplify-font-sizes-medium);
         --amplify-components-badge-large-padding-vertical: var(--amplify-space-small);
         --amplify-components-badge-large-padding-horizontal: var(--amplify-space-medium);
+        --amplify-components-bannermessage-button-background-color: var(--amplify-colors-background-tertiary);
```

Comment:
@cshfang Didn't we decide to namespace these to `--amplify-components-inapp` prefix?